components/modules/Kconfig

```diff
@@ -67,6 +67,12 @@ config LUA_MODULE_BTHCI
   help
       Includes the simple BlueTooth HCI module.
 
+config LUA_MODULE_CAN
```

Comment:
I suppose that the `can` module requires the ESPCAN node. If so, then please establish the related dependencies between this item and the tree in components/driver_can/Kconfig. Something like "depends on" or "selects" from https://github.com/espressif/esp-idf/blob/master/tools/kconfig/kconfig-language.txt#L85.